src/vs/platform/contextkey/browser/contextKeyService.ts

```diff
@@ -46,18 +47,9 @@ export class ContextValuesContainer {
 		}
 		return ret;
 	}
-
-	public fillInContext(bucket: any): void {
-		if (this._parent) {
-			this._parent.fillInContext(bucket);
-		}
-		for (let key in this._value) {
-			bucket[key] = this._value[key];
-		}
-	}
```

Comment:
Hm, it might be this

```ts
public getValue<T>(key: string): T {
		const ret = this._value[key];
		if (typeof ret === 'undefined' && this._parent) {
			return this._parent.getValue<T>(key);
```